src/patternfly/components/DataList/data-list.scss

```diff
@@ -0,0 +1,221 @@
+@import "../../patternfly-utilities";
+
+.pf-c-data-list {
+  --pf-c-data-list--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+
+  // Item
+  --pf-c-data-list__item--PaddingY: var(--pf-global--spacer--xl);
+  --pf-c-data-list__item--PaddingX: var(--pf-global--spacer--xl);
+  --pf-c-data-list__item--380--PaddingX: var(--pf-global--spacer--lg);
+
+  @media screen and (max-width: 380px) {
```

Comment:
Since we use the same max-width property for every MQ, can we create a breakpoint variable for it? And then we could redefine the variable name above, because the "380" in `--pf-c-data-list__item--380--PaddingX` seems a little strange? But correct me if I'm wrong :) Also, if one changes, I assume they all should?

Reply:
@mattnolting what do you think? Is this just a one-off for datalist or should we create a global var?